collect_app/lint.xml

```diff
@@ -1,3 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <lint>
+    <!-- make sure classes referenced in the manifest or in a layout file are in the project -->
+    <issue id="MissingRegistered" severity="error" />
+
+    <!-- require inputType attribute on all text fields -->
+    <issue id="TextFields" severity="error" />
```

Comment:
If we feel that this is something we ought to enforce, we can leave it at ERROR level. My initial feeling was that it is too severe, but thinking about it now, the user experience comes first.